web/frontend/src/mocks/handlers.ts

```diff
@@ -1,13 +1,51 @@
 import { rest } from 'msw';
+import { mockElection1, mockElection2 } from './mockData';
+import ShortUniqueId from 'short-unique-id';
 
 import {
   ENDPOINT_EVOTING_CREATE,
   ENDPOINT_EVOTING_GET_ALL,
+  ENDPOINT_EVOTING_GET_ELECTION,
   ENDPOINT_GET_TEQ_KEY,
   ENDPOINT_LOGOUT,
   ENDPOINT_PERSONNAL_INFO,
 } from '../components/utils/Endpoints';
 
+import { Configuration } from '../components/utils/types';
+
+const uid = new ShortUniqueId({ length: 8 });
+
+interface GetElectionBody {
+  ElectionID: string;
+  Token: string;
+}
+
+interface CreateElectionBody {
+  Title: string;
+  AdminID: string;
+  Token: string;
+  Format: Configuration;
+}
```

Comment:
Those are the interfaces for the request body sent from the frontend client, so shouldn't it be named something like `frontendRequestBody.ts` (since we will add more than the election in the future, like cast ballot get infos etc.)?

Reply:
yes, makes sense 👍